pkg/virt-controller/services/rendervolumes.go

```diff
@@ -54,6 +55,7 @@ func (vr *VolumeRenderer) Mounts() []k8sv1.VolumeMount {
 		mountPathWithPropagation(containerDisks, vr.containerDiskDir, k8sv1.MountPropagationHostToContainer),
 		mountPath("libvirt-runtime", "/var/run/libvirt"),
 		mountPath("sockets", filepath.Join(vr.virtShareDir, "sockets")),
+		mountPath(sriov.VolumeName, sriov.MountPath),
```

Comment:
Note that this mount will be added to every pod.

Reply:
Fixed